Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/magnum.yml

```diff
@@ -0,0 +1,13 @@
+- type: entity
+  noSpawn: true
+  parent: BaseBullet
+  id: Bullet454Casull
+  name: bullet (.454 Casull)
+  components:
+  - type: Projectile
+    damage:
+      types:
+        Piercing: 30
+        Slash: 5
```

Comment:
Not sure if such small difference (compared to regular 45) warrants new ammo type.

Reply:
Well compare this, to the 45 and the 60 and look at the damage spread in between

Reply:
> and the 60 and look at the damage spread in between

How is this relevant? I'm stating that the difference between 'casull' and base game magnum is imperceptible; hence, I'm questioning the need for adding a new type of ammo that does the same thing that magnum does but can only be used by one gun.

Reply:
It is relevant because this was to replace the 60 anti because they said the 60 was too high damage. Also no there will be another gun that would used the 454 casull  but every time I put mulitple guns in a pr I am told to separate them. Thus I haven't PR it yet if you want it I can add both of the other guns to both.